ext/handlers/filters/updates.go

```diff
@@ -0,0 +1,61 @@
+package filters
+
+import (
+	"github.com/PaulSonOfLars/gotgbot/v2"
+)
+
+type ChatMember func(u *gotgbot.ChatMemberUpdated) bool
+
+func (f ChatMember) And(f2 ChatMember) ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return f(cq) && f2(cq)
+	}
+}
+
+func (f ChatMember) Or(f2 ChatMember) ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return f(cq) || f2(cq)
+	}
+}
+
+func (f ChatMember) Not() ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return !f(cq)
+	}
+}
+
+func ChatMemberUserId(id int64) ChatMember {
+	return func(cm *gotgbot.ChatMemberUpdated) bool {
+		return cm.NewChatMember.User.Id == id
+	}
+}
+
+func PerformerUserId(id int64) ChatMember {
+	return func(cm *gotgbot.ChatMemberUpdated) bool {
+		return cm.From.Id == id
+	}
+}
+
+func ChatMemberChatId(id int64) ChatMember {
+	return func(cm *gotgbot.ChatMemberUpdated) bool {
+		return cm.Chat.Id == id
+	}
+}
+
+func ChatMemberAll() ChatMember {
+	return func(_ *gotgbot.ChatMemberUpdated) bool {
+		return true
+	}
+}
+
+func ChatMemberGroup() ChatMember {
```

Comment:
Can we get a channel type here too? Could be useful for determining channel joins.

Reply:
Ok and message filters doesnt have channel too.

Reply:
Feel free to add it then!

Reply:
> Can we get a channel type here too? Could be useful for determining channel joins.

Done